demo-node/entrypoint.bash

```diff
@@ -1,16 +1,27 @@
 #!/usr/bin/env bash
 
-coproc NODE { exec tenzir-node --print-endpoint; }
+export TENZIR_ACTIVE_PARTITION_TIMEOUT="${TENZIR_ACTIVE_PARTITION_TIMEOUT:-5s}"
```

Comment:
I don't think this should be even lower than 30s, this will lead to too small partitions otherwise. I think 30s is fine for an interactive experience.

Reply:
I dropped this line. We can always play around with these variables by setting them in the container runtime.